src/pages/api/index.tsx

```diff
@@ -0,0 +1,266 @@
+// Fetches results from the actor's dataset.
+const { items } = await client.dataset(defaultDatasetId).listItems();
+                                `}/>
+                                    </ClientCodeWrapper>
+                                </SectionWrapper>
+                            ),
+                        },
+                        {
+                            title: <TabTitleWrapper>
+                                <ThemedImage
+                                    height={16}
+                                    width={16}
+                                    sources={{ dark: useBaseUrl('/img/python-40x40.svg'), light: useBaseUrl('/img/python-40x40.svg') }}
+                                />
+                                Python Client
+                            </TabTitleWrapper>,
+                            content: (
+                                <SectionWrapper
+                                    heading="Python API client"
+                                    description={<div className="Description">
+                                        The official library to interact with Apify API from a Python applications.
+                                        <GitHubButton href="https://github.com/apify/apify-client-python" data-color-scheme="no-preference: light; light: light; dark: dark;" data-size="large" data-show-count="true">Star</GitHubButton>
+                                        <div className="DescriptionLinks">
+                                            <Button to='https://docs.apify.com/api/client/python/docs'>Get started</Button>
+                                            <ActionLink to='https://docs.apify.com/api/client/python'>Python client reference</ActionLink>
+                                        </div>
+                                    </div>}
+                                >
+                                    <ClientCodeWrapper>
+                                        <CodeBlock content='pip install apify-client' language="bash"/>
+                                        <CodeBlock
+                                            className="codeBlock"
+                                            language='python'
+                                            hideLineNumbers
+                                            content={`
+from apify_client import ApifyClient
+
+apify_client = ApifyClient('MY-APIFY-TOKEN')
+
+# Start an actor and wait for it to finish
+actor_call = apify_client.actor('john-doe/my-cool-actor').call()
+
+# Fetch results from the actor run's default dataset
+dataset_items = apify_client.dataset(actor_call['defaultDatasetId']).list_items().items
+                                `}/></ClientCodeWrapper>
+                                </SectionWrapper>
+                            ),
+                        },
+                    ]} />
+                </Section>
+                <Section heading="Related articles">
+                    <RelatedArticlesWrapper>
```

Comment:
There should be 2 columns on the tablet based on the design

<img width="953" alt="Snímek obrazovky 2024-05-05 v 12 35 41" src="https://github.com/apify/apify-docs/assets/45016873/20f4e832-cfd1-4112-82a0-09eadfe0ee67">

Reply:
@HonzaTuron now it's 3 columns on the tablet (should be 2)

<img width="1163" alt="Snímek obrazovky 2024-05-07 v 13 20 29" src="https://github.com/apify/apify-docs/assets/45016873/af9630f4-05d8-46d6-9588-eae75538a510">